plugin-cgroupv2/src/cgroupv2.rs

```diff
@@ -1,22 +1,37 @@
-use std::str::FromStr;
-
 use alumet::{
     metrics::{MetricCreationError, TypedMetricId},
     plugin::AlumetPluginStart,
     units::{PrefixedUnit, Unit},
 };
+use anyhow::{Context, Result};
+use std::str::FromStr;
 
 pub(crate) const CGROUP_MAX_TIME_COUNTER: u64 = u64::MAX;
 
 #[derive(Debug, PartialEq, Clone)]
 pub struct CgroupV2Metric {
```

Comment:
Since tests can be boring, forgive me for giving you modifications beyond that :)

`CgroupV2Metric` is not a metric and does not contain metrics. Rename it to `CgroupMeasurements` or something like that.